e2e/browser/testApp/package.json

```diff
@@ -10,17 +10,17 @@
   },
   "dependencies": {
     "@inrupt/solid-client-authn-browser": "^1.11.7",
-    "@inrupt/solid-client-notifications": "*",
-    "next": "12.1.2",
+    "@inrupt/solid-client-notifications": "file:../../../",
```

Comment:
@NSeydoux I switched this to use `file:` instead, after the discussion with the npm team here: https://github.com/npm/feedback/discussions/667